### ISSUE TITLE
no `VectorOfVectorOfPoint` in `types`

### ISSUE THREAD

**Issue by Xuxue1** [twistedfall/opencv-rust#306]
When use opencv set default-features=false, I get a error when use opencv::types::VectorOfVectorOfPoint
1. ubuntu 20.04
2. manual compilation
3. 4.5
4. 1.56
5. opencv = {version= "0.60", default-features=false, features=["clang-runtime", "imgcodecs", "imgproc"]}
   ```shell script
   use opencv::types::VectorOfVectorOfPoint;
  |     ^^^^^^^^^^^^^^^---------------------
  |     |              |
  |     |              help: a similar name exists in the module: `VectorOfVectorOfPoint2f`
  |     no `VectorOfVectorOfPoint` in `types`
   ```

**Comment by twistedfall** [comment]
Can you please tell me why you need the `VectorOfVectorOfPoint` type specifically?

**Comment by Xuxue1** [comment]
opencv::imgproc::find_contours_with_hierarchy need VectorOfVectorOfPoint

**Comment by twistedfall** [comment]
Thanks, it should be fixed in v0.61.1
